### PR TITLE
Add release note for kubedb-v2023.02.28

### PR DESCRIPTION
**Pull request by heheh13**

### CODE REVIEW

File: content/post/kubedb-v2023.02.28/index.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        resources:
+          requests:
+            storage: 10Gi
+      terminationPolicy: Delete
+```
+
+## Redis
+Latest versions for Redis  `6.0.18`, `6.2.11` and `7.0.9` also added  in this release. Example of a  Redis Standalone instance with version `Redis 6.0.18`
+
+```bash
```
Comment (TasdidurRahman) [inline content/post/kubedb-v2023.02.28/index.md:146]: yaml

File: content/post/kubedb-v2023.02.28/index.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+      version: 6.0.18
+      storageType: Durable
+      storage:
+        storageClassName: "standard"
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: 1Gi
+      terminationPolicy: WipeOut
```
Comment (TasdidurRahman) [inline content/post/kubedb-v2023.02.28/index.md:162]: fix indentation

File: content/post/kubedb-v2023.02.28/index.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+       mode: GroupReplication
+      storageType: Durable
+      storage:
+        storageClassName: "standard"
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 10Gi
+      terminationPolicy: Delete
```
Comment (TasdidurRahman) [inline content/post/kubedb-v2023.02.28/index.md:140]: fix indentation

File: content/post/kubedb-v2023.02.28/index.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+```yaml
+apiVersion: ops.kubedb.com/v1alpha1
+kind: ElasticsearchOpsRequest
+metadata:
+  name: os-upgrade
+  namespace: demo
+spec:
+  type: Upgrade
+  databaseRef:
+    name: os-cluster                             # refer the name of you opensearch instance
```
Comment (TasdidurRahman) [inline content/post/kubedb-v2023.02.28/index.md:96]: redundant space. should be removed

File: content/post/kubedb-v2023.02.28/index.md [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+```yaml
+apiVersion: dashboard.kubedb.com/v1alpha1
+kind: ElasticsearchDashboard
+metadata:
+  name: os-cluster-dashboard
+  namespace: demo
+spec:
+  enableSSL: true
+  databaseRef:
+    name: os-cluster                   # refer the name of you opensearch instance
```
Comment (TasdidurRahman) [inline content/post/kubedb-v2023.02.28/index.md:114]: redundant space. should be removed

File: content/post/kubedb-v2023.02.28/index.md [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+spec:
+  type: Upgrade
+  databaseRef:
+    name: os-cluster                             # refer the name of you opensearch instance
+  upgrade:
+    targetVersion: opensearch-2.0.1    # refer the target version to be upgraded 
+```
+We recommend upgrading to OpenSearch `2.0.1` before upgrading to any other V2 minor versions.
+
+### Dashboard
```
Comment (TasdidurRahman) [inline content/post/kubedb-v2023.02.28/index.md:102]: Opensearch Dashboards ?

File: content/post/kubedb-v2023.02.28/index.md [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+  horizontalScaling:
+    replicas: 3
+    streamingMode: Synchronous
+    standbyMode: Hot
+  databaseRef:
+    name: demo-pg
+```
+
+## ElasticSearch
+Lastest versions for  OpenSearch  `2.0.1` and `2.5.0` added in this release. Now, you can provision and manage `OpenSearch V2` with `Elasticsearch` CRD using KubeDB.
```
Comment (TasdidurRahman) [inline content/post/kubedb-v2023.02.28/index.md:73]: ### Latest version support ?

Comment (TasdidurRahman) [reply content/post/kubedb-v2023.02.28/index.md:73]: lastest -> latest

File: content/post/kubedb-v2023.02.28/index.md (hunk 6 above)
Comment (TasdidurRahman) [reply content/post/kubedb-v2023.02.28/index.md:102]: either add heading with ### Latest version release/support or remove the heading for dashboard

File: content/post/kubedb-v2023.02.28/index.md [hunk 8]
```diff
@@ -0,0 +1,181 @@
...
+- kubedb
+- kubernetes
+- mariadb
+- mongodb
+- mysql
+- opensearch
+- percona
+- percona-xtradb
+- postgresql
+- redis
```
Comment (TasdidurRahman) [inline content/post/kubedb-v2023.02.28/index.md:23]: add proxysql ?

**Review by TasdidurRahman — CHANGES_REQUESTED** [review]

**Review by TasdidurRahman — APPROVED** [review]
LGTM

**Review by raihankhan — APPROVED** [review]
LGTM!

**Review by TasdidurRahman — APPROVED** [review]
LGTM

**Pull request merged**